scripts/BI-115.ts

```diff
@@ -0,0 +1,49 @@
+
+import { Agenda } from '../modules/turnos/schemas/agenda';
+import { IPrestacion } from '../modules/rup/prestaciones.interface';
+import { Prestacion } from '../modules/rup/schemas/prestacion';
+import { Profesional } from '../core/tm/schemas/profesional';
+import moment = require('moment');
+
+async function run(done) {
+    const query: any = {
+        $and: [
+            { 'solicitud.turno': { $exists: true } },
+            { 'solicitud.turno' : { $ne: null } }
+        ],
+        'estadoActual.tipo': 'validada'
+    };
+
+    if (process.argv.length >= 3) {
```

Comment:
```suggestion
    if (process.argv.length > 3) {
```
Si es mayor o igual, si no le mandas ningun argumento entra al if porque el length es 3